projects/pdks.yml

```
@@ -0,0 +1,53 @@
+# Project name
+name: Open source PDKs & Workgroup
+# Project status - sandbox, graduated or archived
+status: sandbox
+# Links to project repos - may be one or list of several in YAML list format
+repositories:
+  - https://github.com/google/skywater-pdk
+  - https://github.com/google/gf180mcu-pdk
+# A one-sentence summary of the project
+brief_summary: Open source process design kits for the SkyWater Technology 130nm and GlobalFoundries 180MCU manufacturing processes. These GitHub repos are currently maintained by Google, but proposing to move them into CHIPS Alliance and to form a new workgroup for governance moving forward.
+# The project's open source license, normally Apache 2.0 unless special exception is granted
+license: Apache-2.0
+# Link to issue tracker
+issue_tracker_link: https://github.com/google/skywater-pdk/issues
+# Link to website
+website_link: N/A
```

with https://github.com/chipsalliance/chips-alliance-website/pull/103 merged we can have:

```suggestion
website_link:
  - https://skywater-pdk.readthedocs.io/en/main/
  - https://gf180mcu-pdk.readthedocs.io/en/latest/
```